src/models/validators/ActivityCreated.ts

```diff
@@ -3,7 +3,7 @@ import { ActivityType } from '@dvsa/cvs-type-definitions/types/v1/enums/activity
 import { WaitReason } from '@dvsa/cvs-type-definitions/types/v1/enums/waitReason.enum';
 import { TestStationTypes } from '@dvsa/cvs-type-definitions/types/v1/enums/testStationType.enum';
 
-export const ActivityModel = Joi.object().keys({
+export const ActivityCreated = Joi.object().keys({
```

Comment:
CreateActivity or something like ActivityCreate I think reads better

Reply:
Test results is XCancelled, YSubmitted, etc. Are we not were keeping same pattern?

Reply:
Fair point